docs/msync.md

```diff
@@ -0,0 +1,23 @@
+## TheVoxFox Usage: !msync
+
+TheVoxFox accept a small range of arguments that you can use to retrieve modulesync version statues or generate reports.
+
+Using `!msync` with no args will retrieve the latest semver tag from our modulesync_config repo.
```

Comment:
should we link to https://github.com/voxpupuli/modulesync_config (if somebody is new to our infra, me maybe doesn't know the link?)